webpack.config.js

```diff
@@ -5,7 +5,7 @@ var webpack = require('webpack');
 
 var base = {
     devServer: {
-        contentBase: path.resolve(__dirname, 'playground'),
+        contentBase: false,
```

Comment:
Try this:
- Check out a version of `scratch-vm` from before this change
- Remove `dist` entirely -- this means `scratch-vm.js` must be provided from in-memory build output, not from `contentBase`
- `npm run start`
- Try loading various different paths, such as `/`, `/playground`, `/scratch-vm.js`, and `/playground/scratch-vm.js`.

From my tests while working on this change, I basically became convinced that it has always been broken. I also couldn't find a way to make it provide the build output from `webpack-dev-server` as `/scratch-vm.js` instead of `/playground/scratch-vm.js` -- if you find a way to do that I would be all for it.